sparkler-api/src/main/java/edu/usc/irds/sparkler/util/FetcherDefault.java

```diff
@@ -39,6 +68,15 @@
     public FetchedData fetch(Resource resource) throws Exception {
         LOG.info("DEFAULT FETCHER {}", resource.getUrl());
         URLConnection urlConn = new URL(resource.getUrl()).openConnection();
+        if (httpHeaders != null){
+            httpHeaders.entrySet().forEach(e -> urlConn.setRequestProperty(e.getKey(), e.getValue()));
+        }
+        String userAgentValue = getUserAgent();
+        if (getUserAgent() != null) {
+            LOG.debug(USER_AGENT + ": " + userAgentValue);
+            urlConn.setRequestProperty(USER_AGENT, userAgentValue);
```

Comment:
Okay, let's go with overriding the default `User-Agent` but do you see worth in the following?
(a) Making this a set than list to avoid duplicate agents.
(b) Moving `fetcher.user.agents` in a separate file. This will make the maintenance easier. (Un)commenting a single line is better than many lines to enable/disable the agent rotation.